app/models/concerns/foreman_bootdisk/host_ext.rb

```diff
@@ -33,11 +33,17 @@ def bootdisk?
     managed? && bootdisk_build? && SETTINGS[:unattended]
   end
 
+  def bootdisk_downloadable?
+    ['ia32', 'x64'].include?(architecture.try(:bootfilename_efi))
```

Comment:
I am actually not well versed with OS and it's underlying architecture, just know basics. So I am assuming intel doesn't have anything more than those architecture names(i386 for 32 bit and x86_64 is 64 bit)